src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.component.ts

```diff
@@ -26,14 +26,31 @@ import { faBan, faSave } from '@fortawesome/free-solid-svg-icons';
 import { DocumentationType } from 'app/shared/components/documentation-button/documentation-button.component';
 import { scrollToTopOfPage } from 'app/shared/util/utils';
 import { loadCourseExerciseCategories } from 'app/exercises/shared/course-exercises/course-utils';
+import { FormSectionStatus } from 'app/forms/form-status-bar/form-status-bar.component';
+import { Subscription } from 'rxjs';
+import { ExerciseTitleChannelNameComponent } from 'app/exercises/shared/exercise-title-channel-name/exercise-title-channel-name.component';
+import { NgModel } from '@angular/forms';
+import { ExerciseUpdatePlagiarismComponent } from 'app/exercises/shared/plagiarism/exercise-update-plagiarism/exercise-update-plagiarism.component';
+import { TeamConfigFormGroupComponent } from 'app/exercises/shared/team-config-form-group/team-config-form-group.component';
+import { FormDateTimePickerComponent } from 'app/shared/date-time-picker/date-time-picker.component';
 
 @Component({
     selector: 'jhi-modeling-exercise-update',
     templateUrl: './modeling-exercise-update.component.html',
 })
-export class ModelingExerciseUpdateComponent implements OnInit {
+export class ModelingExerciseUpdateComponent implements AfterViewInit, OnDestroy, OnInit {
+    @ViewChild(ExerciseTitleChannelNameComponent) exerciseTitleChannelNameComponent: ExerciseTitleChannelNameComponent;
+    @ViewChild(ExerciseUpdatePlagiarismComponent) exerciseUpdatePlagiarismComponent?: ExerciseUpdatePlagiarismComponent;
+    @ViewChild(TeamConfigFormGroupComponent) teamConfigFormGroupComponent?: TeamConfigFormGroupComponent;
     @ViewChild(ModelingEditorComponent, { static: false })
     modelingEditor?: ModelingEditorComponent;
+    @ViewChild('bonusPoints') bonusPoints?: NgModel;
```

Comment:
Ok, I just checked and that would require a complete overhaul of this component from a template-driven form to reactive forms (I think).
So I would say it would be better to disregard it for this PR, maybe could be adressed in a follow-up if it makes sense?

Reply:
We have been arguing about a refactoring to reactive forms, but since that would be a lot of effort because of custom-non-input components, like the markdown component, we agreed for that to be out of scope for the validation bar.